localisation/00_zuwc_united_localisation.csv

```diff
@@ -303,7 +303,7 @@ EVTOPTB_WCBDP_110_alliance;We have to destroy the Portal!;;Da wir nicht in der L
 EVTDESC_WCBDP_140;[Root.PrimaryTitle.GetFOA], I am sending you with my army beyond the Dark Portal. You have to deal with the danger that comes from beyond.;;[Root.PrimaryTitle.GetFOA], ich sende Euch mit meiner Armee durch das dunkle Portal. L�scht die Gefahren, die von jenseits des Portals kommen, aus.;;;;;;;;;;;x
 EVTTITLE_WCBDP_150;Destruction of Draenor;La destruction de Draenor;Die Zerst�rung von Draenor;;;;;;;;;;;x
-EVTDESC_WCBDP_150;The destruction of the Dark Portal as well as the dark magic of the Shadowmoon clan have done their dark deeds. Draenor, the world of the orcs, dies. There are earthquakes everywhere, the waters fall into the abyss and the streams of fel burn out what was once fertile meadows. This world is about to end and there is no path to salvation.\nIt is unlikely that Azeroth's inhabitants will ever truely know what happened today among the plains of this wild planet, but they will definitely remember the final victory over the invading forces that were hiding on the other side of the portal.\nThere are reports, however, that Ner'zhul, the former shaman, and his clan saved themselves in the Twisting Nether by escaping through one of the portals he created himself.;;Die Zerst�rung des dunklen Portals und die dunkle Magie des Schattenmond-Clans haben ihre finstere Arbeit getan! Draenor, die Heimatwelt der Orks, stirbt! Erdbeben �berall, die Gew�sser flie�en in den Abgrund, Str�me von Felmagie lassen die einst fruchtbaren Wiesen verbrennen. Diese Welt kommt zu einem vorzeitigen Ende, es gibt kein Zur�ck mehr.\nF�r die Horde wird es schwer, herauszufinden, was auf Draenor an diesem schicksalhaften Tag wirklich passiert ist, aber jeder ehrenhafte Ork wird sich an die Valiance-Expedition und ihre heldenhaften Bem�hungen gegen die gr��te und gef�hrlichste Invasion in der Geschichte des neuen Orkreichs erinnern.\nDie Expedition kann noch gerettet werden und durch eines der von Ner'zhul ge�ffneten Portale entkommen, es l�sst sich aber nicht voraussagen, wohin die Portale f�hren. Es scheint, dass der ehemalige Schamane selbst durch eins der Portale gegangen ist und sich selbst und seinen Clan durch den wirbelnden Nether gerettet hat.;;;;;;;;;;;x
+EVTDESC_WCBDP_150;The destruction of the Dark Portal as well as the dark magic of the Shadowmoon clan have done their dark deeds. Draenor, the world of the orcs, dies. There are earthquakes everywhere, the waters fall into the abyss and the streams of fel burn out what was once fertile meadows. This world is about to end and there is no path to salvation.\nIt is unlikely that Azeroth's inhabitants will ever truely know what happened today among the plains of this wild planet, but they will definitely remember the final victory over the invading forces that were hiding on the other side of the portal.\nThere are reports, however, that Ner'zhul, the former shaman, and his clan saved themselves in the Twisting Nether by escaping through one of the portals he created himself.;;;;;;;;;;;;;x
```

Comment:
Please add german "Die Zerstörung des dunklen Portals sowie die dunkle Magie des Shadowmoon-Clans haben ihr dunkles Werk getan. Draenor, die Welt der Orks, stirbt. Überall gibt es Erdbeben, das Wasser fällt in den Abgrund und Ströme von Fel-Energie verbrennen die einst fruchtbaren Wiesen. Diese Welt steht kurz vor dem Ende und es gibt keinen Weg zur Erlösung. Es ist unwahrscheinlich, dass die Bewohner Azeroths jemals genau wissen werden, was heute in den Ebenen dieses wilden Planeten passiert ist, aber sie werden sich definitiv an den endgültigen Sieg über die Invasionstruppen erinnern, die sich auf der anderen Seite des Portals versteckt haben.\n Es gibt jedoch Berichte, dass Ner'zhul, der ehemalige Schamane, und sein Clan sich in den wirbelnden Nether gerettet haben, indem sie durch eines der Portale geflohen sind, die er geschaffen hat."